Javascript/lib/digital_root.js

```diff
@@ -1,4 +1,4 @@
-// Write a method, digital_root(num).
+// Write a method, digitalRoot(number).
```

Comment:
in addition to matching the provided method name of `digitalRoot`, i also changed `num` to `number` to match the parameter name in the provided method signature.